WordPress/Classes/Models/Post.m

```diff
@@ -41,9 +41,9 @@ - (void)didTurnIntoFault
     self.specialType = nil;
 }
 
-- (void)didSave
+- (void)willSave
 {
-    [super didSave];
+    [super willSave];
     if ([self isDeleted]) {
```

Comment:
I'm very sorry about the fuzz in here. As @aerych pointed out, *isDeleted* will in fact be effective here, because it's being called in the *willSave* method (which runs during the *save* OP). Please, disregard my comment, since it's incorrect.

(Sorry Eric!)